assets/js/components/Dashboard/PaMessagesPage.tsx

```diff
@@ -33,12 +33,13 @@ const PaMessagesPage: ComponentType = () => {
                   Add New
                 </Link>
               </Col>
-              <Col className="pa-message-table-action-bar__search">
+              {/* Disabled as these features are not yet implemented */}
```

Comment:
suggestion: Can remove for now because this will not be implemented until after MVP launch.

Reply:
Are you suggesting I reword this comment or that I remove the code completely?